examples/hr360/README.md

```diff
@@ -65,6 +65,71 @@ And add an import for `Skill`, and include it in the `export` section so it look
     workspace
   }
 
+### Adding necessary indexes
```

Comment:
I think the better way to do this with DHF is to first adjust the config paths in gradle.properties so you can easily add your own config:

    mlConfigPaths=src/main/hub-internal-config,src/main/ml-config,src/main/user-config

Then in the user-config dir, add `databases/final-database.json` and add all your config there. It will get merged with the DHF final-database files. Now your config is part of version control and you don't need this README step here.

Reply:
Done and agreed; much cleaner this way.